docs/reference_guide.md

```diff
@@ -1117,6 +1117,18 @@ open path: interrupts
 [...]
 ```
 
+If the traced function is in a kernel module, you can optionally specify it to use BTF data from that
+module's BTF file:
+```
+# cat kvm.bt
```

Comment:
What is kvm.bt?

Reply:
Oh, it's just a made-up name of a file: similar to how the other multi-line examples are in the Reference Guide.

Reply:
Ha, I see them now. I wonder why those exist when they don't point to actual bpftrace scripts in the tools folder. Seem a bit weird to me - maybe I'll make another PR to remove them all 😃